src/pages/ArticlePage.jsx

```diff
@@ -23,6 +29,16 @@ const ArticlePage = () => {
       });
   }, [article_id]);
 
+  const handleVote = (change) => {
+    setVotes(votes + change);
```

Comment:
You should be using the call back function that a  to update the state without mutating it directly (may help with my main comment above. Have a refresh [here in the notes](https://notes.northcoders.com/courses/js-front-end/react-state#setstate-updater-functions)